dat/missions/neutral/adblocker.lua

```diff
@@ -0,0 +1,164 @@
+--[[
+<?xml version='1.0' encoding='utf8'?>
+<mission name="Adblocker">
+ <unique />
+ <chance>20</chance>
+ <location>Bar</location>
+ <faction>Dvaered</faction>
+ <faction>Empire</faction>
+ <faction>Frontier</faction>
+ <faction>Goddard</faction>
+ <faction>Independent</faction>
+ <faction>Sirius</faction>
+ <faction>Soromid</faction>
+ <faction>Za'lek</faction>
+</mission>
+--]]
+--[[
+    Adblocker
+    Author: Lemuria
+
+    Player has to kill a ship that's been spamming the local comms with
+    excessive advertisements.
+
+   I was bored and had nothing better to do, so I made this Naev mission,
+   my first.
+--]]
+
+local fmt = require "format"
+local neu = require 'common.neutral'
+local vn = require "vn"
+local vntk = require "vntk"
+local portrait = require "portrait"
+
+local mission = {
+   name = _("Adblocker"),
+   description = "Destroy a ship spamming the {sys} system with excessive advertising transmissions",
+   reward = 300e3,
+   npc = {
+      name = "Desperate captain",
+      img = "neutral/male7.webp",
+      description = _("You see a desperate looking captain.")
+   }
+}
+
+-- fp means "full path"
+mission.npc["img_fp"] = portrait.getFullPath(mission.npc.img)
+
+function create()
+   mem.current_system = system.cur()
```

Comment:
You should do an inclusive claim in the current system, or the pilot you create could get erased by another mission.